style.css

```diff
@@ -80,30 +80,46 @@ nav {
   display: none;
 }
 
-.nav-menu {
+.blurred-bg {
   background: #6070ff;
   position: fixed;
+  top: 0.0000001px;
+  left: 0.0000001px;
+  width: 100%;
+  height: 100%;
+  opacity: 0.9;
+  filter: blur(5px);
+}
+
+.nav-menu {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  top: 0.0000001px;
+  left: 0.0000001px;
   width: 100%;
   height: 100%;
-  top: 0.000000001px;
-  left: 0.000000001px;
-  right: 0.000000001px;
-  opacity: 0.98;
-  filter: blur(8px);
```

Comment:
- [ ] [optional] Practically the navbar should be attached with the top edge of the viewport, currently there is an empty space between the top edge of the navbar and the viweport, would you fix that?

Reply:
When I make the top value zero, there is a linter error. But I will try that again. Thank you.